### ISSUE TITLE
Failed to fetch Sample from MONAI Label Server

### ISSUE THREAD

**Issue by dyollb** [Project-MONAI/MONAILabel#859]
### Discussed in https://github.com/Project-MONAI/MONAILabel/discussions/858

<div type='discussions-op-text'>

<sup>Originally posted by **dyollb** July  1, 2022</sup>
I am trying MONAILabel for the first time.

## I put my images in following hierarchy (I have about 60 images, and 3 labeled images, single label):
```
- F:\Data\eyes_MONAILabel
  - labels
    - final
      - prefix_001.nii.gz
      - prefix_002.nii.gz
      - prefix_003.nii.gz
  - prefix_001.nii.gz
  - prefix_002.nii.gz
  - prefix_003.nii.gz
  - prefix_004.nii.gz
  - prefix_005.nii.gz
  - ...
```

## I installed
- slicer preview 5.1.0-2022-06-28, and MONAI plugin
- MONAILabel current main branch (`pip install monai-label` did not work, when I entered the URL in Slicer I got an error)
  - git clone https://github.com/Project-MONAI/MONAILabel
  - pip install torch torchvision --extra-index-url https://download.pytorch.org/whl/cu113
  - pip install -e MONAILabel

## Starting the server
- first downloed the app
- `monailabel apps --download --name radiology --output apps`
- the start the server
- `monailabel start_server --app apps/radiology --studies "F:\Data\eyes_MONAILabel" --conf models deepedit`

## The problem I am having
- I could start the plugin and enter the URL. 
- Then I ran `Train` until it reached 100%
- Then I pressed `Nex Sample`
![image](https://user-images.githubusercontent.com/12702862/176842142-a7b8f468-3302-4e81-b852-61bb2a80f8b7.png)

Since the error pop-up cannot be resized I scrolled down and show the rest of the callstack:
![image](https://user-images.githubusercontent.com/12702862/176843948-c6d9bfbe-e208-41f9-b243-36eea69d0712.png)

Is this due to incompatible Slicer / Slicer plugin / MONAILabel app / MONAILabel package versions?</div>

**Comment by dyollb** [comment]
I see now that the error message is not in "MONAILabel.py" from my installation of MONAI, but instead from the Slicer installation... Mmmmh

**Comment by diazandr3s** [comment]
Thanks for the detailed information, @dyollb.
If it is a single-label task, please change the label name [here](https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/radiology/lib/configs/deepedit.py#L46-L55), if using deepedit, or [here](https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/radiology/lib/configs/segmentation.py#L33-L47) if using segmentation model.
I also suggest you set [this flag](https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/radiology/lib/configs/segmentation.py#L59) to false so you don't use the default pretrained model. That pretrained model is for multilabel - it doesn't work for a single label task

More information is presented here: https://youtu.be/wtiEe_jiUzg?t=2841

Hope this helps

**Comment by dyollb** [comment]
@diazandr3s, @lassoan 
- the error in MONAILabel.py says qMRMLSegmentEditorWidget has no attribute setMasterVolumeNode
- the error is from `C:\Users\lloyd\AppData\Local\NA-MIC\Slicer 5.1.0-2022-06-28\NA-MIC\Extensions-31073\MONAILabel\lib\Slicer-5.1\qt-scripted-modules\MONAILabel.py`
- why is it not using the MONAILabel.py I installed from the github repo via `git clone https://github.com/Project-MONAI/MONAILabel; pip install -e MONAILabel` 
-

**Comment by jlvahldiek** [comment]
I can confirm this, today I encountered the same error with Slicer 5.1.0-2022-06-28. Downgrading to 5.0.2 temporarily solved the problem.

**Comment by lassoan** [comment]
Sorry for the complications, it was a regression in Slicer Preview Release (Slicer-5.1.x) introduced a few days ago that we have fixed. MONAILabel will work well in Slicer-5.1.x releases downloaded tomorrow or later.

**Comment by SachidanandAlle** [comment]
closing this for now.. please reopen if the issue still persists
